src/sdgio-edit.owl

```diff
@@ -3291,59 +3292,48 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
 AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020297> "enhanced individual capacity in partner research organizations through training and exchange"@en)
 ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020237> <http://purl.unep.org/sdg/SDGIO_00020297>)
 
-# Individual: <http://purl.unep.org/sdg/SDGIO_00020298> (increased capacity for innovations in partner research organizations)
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020298> (increment of the capacity for innovations in partner research organizations)
```

Comment:
Revise to:

"increase the capacity for innovation in research organizations partnered with the CGIAR"